templates/cmd/controller/main.go.tpl

```diff
@@ -36,6 +36,14 @@ func main() {
 	flag.Parse()
 	ackCfg.SetupLogger()
 
+	if ackCfg.AccountID == "" {
+    	setupLog.Error(
+    		errors.New("Unable to start service controller as account ID is nil. Please pass --aws-account-id flag"), "unable to create controller manager",
+    		"aws.service", awsServiceAlias,
+    	)
+    	os.Exit(1)
+    }
```

Comment:
We should not place validation code for particular configuration flags into the template here. Instead, make a call to a `ackCfg.Validate()`:

```
if err := ackCfg.Validate(); err != nil {
    setupLog.Error(...)
    os.Exit(1)
}
```

and then simply implement the `ackrt.Config.Validate()` method to check its configuration flag values as needed. That way, we keep all implementation specifics in the `ackrt.Config.Validate` method and we don't need to change the template file each time we add a new configuration validation.
```